release-notes/release-notes503.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: default
+title: TinyMCE 5.0.3
+title_nav: TinyMCE 5.0.3
+keywords: releasenotes newfeatures deleted technologypreview bugfixes knownissues
+---
+
+## Overview
+
+The Release Notes provide high-level coverage of the improvements and additions that have been implemented in TinyMCE 5.0.3 and document known problems in this release, as well as notable bug fixes, deprecated functionality, and other details.
+
+TinyMCE 5.0.3 release adds improvements to the TinyMCE editor to improve the overall user experience.
+
+## New features
+
+### Case Change
+
+TinyMCE 5.0.3 introduces the **Case Change** plugin that allows changing the case of text in block selection to uppercase, lowercase, or title case.
+
+Currently, there are three types of formats available:
+
+![**Case Change**]({{site.baseurl}}/images/casechange.png)
+
+* **lowercase** - The **lowercase** option changes all the selected characters to lowercase.
+
+* **UPPERCASE** - The **UPPERCASE** option changes all the selected characters to uppercase.
+
+* **Title Case** - The **Title Case** option changes anything other than articles, coordinating conjunctions, or short prepositions to capital letters. The default rule set for **Title Case** is based on [Wikipedia Title Case](https://titlecaseconverter.com/rules/#WP).
+
+To try out **Case Change** start with this [dedicated product page](https://apps.tiny.cloud/products/case-change/) to see all the flexible pricing options.
+
+For more information on **Case Change** refer to the full [documentation]({{site.baseurl}}/plugins/casechange/).
+
+There is also a demo provided to explore the **Case Change** capabilities [here]({{site.baseurl}}/demo/casechange/).
+
+### TinyMCE for Swing integration
+
+Users can easily configure the TinyMCE editor in Swing through the **TinyMCE for Swing** integration.
```

Comment:
We might want to make this "...in Java's Swing framework..." or something, just to clarify? I don't _think_ there's any other programming things called Swing, but it can't hurt. Also gets the Java keyword in there ;)

Reply:
I initially had some reference text there, but the reviewers thought this was common knowledge. Leaving it as is here. Will update if more reference is required in the future.